### PR TITLE
Fix syntax in documentation example

### PR DESCRIPTION
**Pull request by jorgheymans**

### CODE REVIEW

File: site/src/pages/docs/client-decorator.mdx [hunk 1]
```diff
@@ -42,14 +42,17 @@ import com.linecorp.armeria.client.HttpClient;
...
     public AuditClient(HttpClient delegate) {
         super(delegate);
+        this.delegate = delegate;
     }
 
     @Override
     public HttpResponse execute(ClientRequestContext ctx, HttpRequest req) throws Exception {
         auditRequest(req);
-        return delegate().execute(ctx, req);
+        return delegate.execute(ctx, req);
```
Comment (minwoox) [inline site/src/pages/docs/client-decorator.mdx:55]: Could you just revert the change and use `unwrap()` instead?

Comment (minwoox) [reply site/src/pages/docs/client-decorator.mdx:55]: Could you also fix the code in server-decorator.mdx, please?

**Comment by trustin** [comment]
Thanks a lot for your first contribution, @jorgheymans !
